utils/scripts/create-css-variables.tsx

```diff
@@ -70,8 +70,11 @@ type UvThemeType = {
   breakpoints: Record<string, string>
 }
 
-export const generateThemeCss = (uvTheme: UvThemeType) =>
-  `:root {\n${
+export const generateThemeCss = ({
+  uvTheme,
+  filename,
+}: { uvTheme: UvThemeType; filename: string }) =>
+  `:root.${filename}-theme {\n${
```

Comment:
You need to generate the theme locally and push it using `pnpm run tokens:update`